source/common/stats/thread_local_store.cc

```diff
@@ -242,14 +245,21 @@ void ThreadLocalStoreImpl::releaseScopeCrossThread(ScopeImpl* scope) {
   }
 }
 
+void ThreadLocalStoreImpl::TlsCache::eraseScope(uint64_t scope_id) { scope_cache_.erase(scope_id); }
```

Comment:
@mattklein123 RE closing this PR -- we could, but I was wondering what you thought about submitting the changes around this comment which I think make it easier to debug if there's an issue in this code in the future.

Reply:
I'm not sure what changes you are referring to. Can you maybe open a fresh PR?

Reply:
I think we've now done most of what I wanted here. closing.